lib/Sema/MiscDiagnostics.cpp

```diff
@@ -1964,6 +1970,26 @@ bool swift::fixItOverrideDeclarationTypes(InFlightDiagnostic &diag,
 
   llvm_unreachable("unknown overridable member");
 }
+};
+
+bool swift::computeFixitsForOverridenDeclaration(
+    ValueDecl *decl, const ValueDecl *base,
+    llvm::function_ref<Optional<InFlightDiagnostic>(bool)> diag) {
+  SmallVector<std::tuple<NoteKind_t, SourceRange, std::string>, 4> Notes;
+  bool hasNotes = ::fixItOverrideDeclarationTypesImpl(decl, base, Notes);
+
+  Optional<InFlightDiagnostic> diagnostic = diag(hasNotes);
+  if (!diagnostic) return hasNotes;
+
+  for (const auto &note : Notes) {
+    if (std::get<0>(note) == FixItReplace) {
+      diagnostic->fixItReplace(std::get<1>(note), std::get<2>(note));
+    } else {
+      diagnostic->fixItInsert(std::get<1>(note).Start, std::get<2>(note));
+    }
```

Comment:
I have spared you all from having to read a senselessly complex `std::apply` instead of a few branches and `get`s.

You're welcome.